README.md

```diff
@@ -39,15 +39,18 @@ $ npm install tap
 
 ```console
 $ make runtimetest validation-executables
+$ sudo make RUNTIME=runc localvalidation
```

Comment:
Do we need to add the following? 

``
for EXECUTABLE in runtimetest validation/linux_rootfs_propagation_shared.t validation/create.t validation/linux_gid_mappings.t validation/linux_rootfs_propagation_unbindable.t validation/linux_readonly_paths.t validation/linux_masked_paths.t validation/mounts.t validation/process.t validation/root_readonly_false.t validation/linux_sysctl.t validation/default.t validation/linux_devices.t validation/process_capabilities.t validation/process_oom_score_adj.t validation/process_rlimits.t validation/root_readonly_true.t validation/hostname.t validation/linux_uid_mappings.t; \
do \
	test -x "${EXECUTABLE}"; \
done || (echo "missing test executable; run 'make runtimetest validation-executables'" >&2 && exit 1)
``

The other looks good to me.

Reply:
> Do we need to add the following?
>
>     for EXECUTABLE
>     …

I've pushed 07a3809 → 23f547a to remove that noise from the `localvalidation` output.